hack/install_catatonit.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+CATATONIT_PATH="/usr/bin/catatonit"
+CATATONIT_VERSION="v0.1.3"
+
+if [ -f /usr/bin/catatonit ]; then
+	echo "skipping ... catatonit is already installed"
+else
+	echo "downloading catatonit to $CATATONIT_PATH"
+	curl -o catatonit -L https://github.com/openSUSE/catatonit/releases/download/$CATATONIT_VERSION/catatonit.x86_64
```

Comment:
I am not sure that this is going to cut it, when this package is built in koji, which blocks pulling from the internet.  I think we will need to vendor the package in somehow.

Reply:
It's worth mentioning that this will only be executed when running the integration tests. It's not part of building etc. Hence, I don't see a problem unless Koji does something I am not aware of.

Reply:
Ok, then catatonic will have to come from a different package then podman.